### DISCUSSION TITLE
When will DIVE get released for the GoPro 12?

### DISCUSSION THREAD

**Discussion by sabonev** [gopro/labs#1163] [Feature Requests]
What is the plan on releasing new features for the 12? or we should not expect that anymore?

**Comment by dnewman-gpsw** [comment]
Looking like next week. Labs gets an update with every main camera release, which just happen for HERO12 on Tuesday. QA is finishing up on HERO12 Labs 2.30.70 now, which includes DIVE.

**Comment by Rabii-Hayani** [comment]
Yes new release will be coming because the new firmware update 2.30 included 

HERO12 Black v2.30

USABILITY IMPROVEMENTS

Optimizes performance for longer runtimes when shooting in hot conditions.
Improves orientation detection when recording with QuikCapture.
BUG FIX

Fixes an issue where The Remote was unable to connect to multiple cameras after wake-up.

**Comment by JiDey0007** [comment]
This Tuesday or the next?

**Reply by dnewman-gpsw** [reply]
Main line was last Tuesday, Labs version should be today.

**Reply by dnewman-gpsw** [reply]
Live now. Lot in this one, mostly Labs parity with HERO13.

**Reply by Rabii-Hayani** [reply]
Where is the history of whats new in this version.of hero 12 black labs firmware 
We need a release note to see whats new ?????

**Reply by dnewman-gpsw** [reply]
In the release notes, posted at the same time, same location.  https://gopro.github.io/labs/control/notes/

**Reply by JiDey0007** [reply]
thank's ! Installed, everything works, 12 and 13 are ready

**Reply by photodavo** [reply]
If I add DIVE=1 to a QR code for my 12 the camera beeps even with Beep Volume set to 0%,  Applying a QR code with the same settings without DIVE=1 does not beep.  Is this expected behavior?

**Reply by dnewman-gpsw** [reply]
Not expected, share your command, as you likely have an error.

**Reply by photodavo** [reply]
I believe I found the problem with my command - did not realize I needed the dollar sign in front of the FourCC code to indicate temporary.  When I changed the command to $DIVE=1 in the QR Code Creator and left all other settings the same the beep goes away.

**Reply by dnewman-gpsw** [reply]
That was my guess ;)
